express/parsers/structure.py

```diff
@@ -41,6 +68,89 @@ def __init__(self, *args, **kwargs):
         self.lattice_only_structure = mg.Structure.from_str(self.structure_string, self.structure_format)  # deepcopy
         self.lattice_only_structure.remove_sites(range(1, len(self.structure.sites)))
 
+        # testing
+
+    def get_inchi_run(self):
+        return inchi_run
+
+    def get_inchi_null(self):
```

Comment:
Null values if pybel/openbabel import fails.

Reply:
We don't really need a separate function for just returning null, do we?

Reply:
not necessarily, i can combine functions.